.releaserc.json

```diff
@@ -3,7 +3,8 @@
     "@semantic-release/commit-analyzer",
     "@semantic-release/release-notes-generator",
     ["@semantic-release/npm", { "npmPublish": true }],
-    "@semantic-release/changelog"
+    "@semantic-release/changelog",
```

Comment:
I think you can remove this as it only generates a `CHANGELOG.md`?

Reply:
@lukashass I removed, but I am still not quite sure which plugins are needed and which not. `semantic-release/release-notes-generator` will then generate the release entry with changes for the `github` plugin? We will see with the next release ...